source/api/presentation/3.0/index.md

```diff
@@ -110,49 +117,51 @@ The overall description of the structure and properties of the digital represent
 
 A virtual container that represents a particular view of the object and has content resources associated with it or with parts of it. The Canvas provides a frame of reference for the layout of the content, both spatially and temporally. The concept of a Canvas is borrowed from standards like PDF and HTML, or applications like Photoshop and PowerPoint, where an initially blank display surface has images, video, text and other content "painted" on to it.
 
-##### Content
-{: #overview-content}
+##### Range
+{: #overview-range}
 
-Content resources such as images, audio, video or text that are associated with a Canvas.
+An ordered list of Canvases, and/or further Ranges. Ranges allow Canvases, or parts thereof, to be grouped together in some way. This could be for content-based reasons, such as might be described in a table of contents or the set of scenes in a play. Equally, physical features might be important such as quires or gatherings, or when recorded music is split across different physical carriers such as two CDs.
 
 ### 2.2. Additional Types
 
-##### Collection
-{: #overview-collection}
+This specification makes use of the following types, defined elsewhere:
 
-An ordered list of Manifests, and/or further Collections. Collections allow easy navigation among the Manifests in a hierarchical structure, potentially each with its own descriptive information. Collections might be used to model dynamic result sets from a search, fixed sets of related resources, or other groupings of Manifests for presentation.
+##### Annotation Collection
+{: #overview-annotationcollection}
+
+An ordered list of Annotation Pages. Annotation Collections allow higher level groupings of Annotations to be recorded. For example, all of the English translation Annotations of a medieval French document could be kept separate from the transcription or an edition in modern French, or the director's commentary on a film can be separated from the script.
 
 ##### Annotation Page
 {: #overview-annotationpage}
 
-An ordered list of Annotations in a single response, typically associated with a single Canvas, and can be part of an Annotation Collection.
+An ordered list of Annotations, typically associated with a Canvas but may be referenced from other resource types as well. Annotation Pages are often part of an Annotation Collection.
```

Comment:
The lack of a verb before 'but' bothers me.  "An ordered list of Annotations, it is typically associated with a Canvas but may ..."

Reply:
:+1: Fixed.